src/steamship/data/tags/tag_constants.py

```diff
@@ -87,3 +100,69 @@ def from_html_tag(tagname: Optional[str]) -> Optional[str]:  # noqa: C901
             return DocTag.teletype
 
         return None
+
+
+class TokenTag:
+    """A set of `name` constants for Tags with a `kind` of `TagKind.token`; appropriate for parsing-level ideas."""
+
+    TextWithWs = "textWithWs"
```

Comment:
TextWithWs, like.... "Where's Waldo? He's out Wandering" = true; "The magician made the letter that came before x and after v disappear" = false.

Just being silly - I assume this is whitespace - it should be spelled out.

Reply:
😆  "W" finding is big business!

(yeah it's whitespace)